src/model_repository_manager.cc

```diff
@@ -256,6 +256,11 @@ GetModifiedTime(const std::string& path)
   }
 
   for (const auto& child : contents) {
+    if (child.empty()) {
```

Comment:
Shouldn't we add this test in `GetDirectoryContents` implementation? and directly return an error from `GetDirectoryContents` function? 
This way this failing case will be handled by all the callers of this function.

Reply:
Yes, thanks for pointing out. Moved the check into each of the three cloud storage implementation. [Move check into GetDirectoryContents](https://github.com/triton-inference-server/core/pull/179/commits/2ac27283e63b4222926df29bec06ee5b08fad766)